src/components/createElementComponent.tsx

```diff
@@ -208,15 +165,7 @@ const createElementComponent = (
 
   // Only render the Element wrapper in a server environment.
   const ServerElement: FunctionComponent<PrivateElementProps> = (props) => {
-    // Validate that we are in the right context by calling useElementsContextWithUseCase.
-    const ctx = useElementsOrCustomCheckoutSdkContextWithUseCase(
-      `mounts <${displayName}>`
-    );
-
-    useCartElementContextWithUseCase(
-      `mounts <${displayName}>`,
-      'customCheckoutSdk' in ctx
-    );
+    useElementsOrCustomCheckoutSdkContextWithUseCase(`mounts <${displayName}>`);
```

Comment:
@pololi-stripe mind double-checking my change here and ensure Custom Checkout is still going to be happy?